### DISCUSSION TITLE
Add formatting function for DataLabels plugin

### DISCUSSION THREAD

**Discussion by soulfire03** [erossini/BlazorChartjs#80] [Ideas]
Hello, 
Thank you for porting this library to blazor! 

In the original library there is a "formatter" method property in which we can pass a function, like you did for the tooltip callbacks.
Could you add this please?

**Comment by yhbder** [comment]
I’ve been working on implementing this feature("formatter") but have encountered several challenges. Despite multiple attempts, I haven't been able to make it work. Could you please provide guidance or suggest an alternative approach to help me resolve this issue?
